sdk/python/generator/AbstractImplMap.py

```diff
@@ -30,4 +30,4 @@ def add(self, parent_type: StructObject, child_type: StructObject):
 		self.mapping[parent_type.typename]['children'].append(child_type)
 
 	def get(self, name):
-		return self.mapping[name]
+		return self.mapping.get(name, None)
```

Comment:
i was trying to generate just code for block.cats, which includes transaction.cats (but no concrete transactions). 
this works around that. 

i left changes because i thought it would be nice if generator would not crash in that case, but could revert as we don't _really_ expect that anywhere.